fedora_cloud_base/1.0.0/actions/run.sh

```diff
@@ -1 +1,2 @@
 sudo dnf install -y python python-dnf ansible libselinux-python
+sudo /usr/sbin/setenforce 0 
```

Comment:
You should set it also for permanent in /etc/sysconfig/selinux BUT, let's not do it now, we need to migrate it from FC to Ubuntu (or different stuff)